integrations/extensions/starter-kits/google-custom-search/README_TEMPLATE.md

```diff
@@ -0,0 +1,62 @@
+
+The OpenAPI spec used by the extension includes the following endpoints:
+
+- `GET /customsearch/v1`: Search for content relevant to a given query over the entire web.
+- `GET /customsearch/v1/siterestrict`: Search for content relevant to a given query over a specific collection of websites.
+
+The endpoints are described in detail at:
+
+1. **Custom Search**: https://developers.google.com/custom-search/v1/reference/rest/v1/cse/list
+2. **Custom Search Site Restricted**: https://developers.google.com/custom-search/v1/reference/rest/v1/cse.siterestrict/list
+
+These endpoints have the same arguments and responses.  However, there are two major differences between these endpoints:
+
+- The *Custom Search Site Restricted* endpoint is restricted to searching 10 or fewer websites (each of which can have an unlimited number of pages).  In contrast, *Custom Search* can support any number of websites including all of the web that is indexed by Google.
+- The *Custom Search* endpoint has a [daily query limit](https://developers.google.com/custom-search/v1/overview#pricing), and the *Custom Search Site Restricted* does not.
+
+For a typical assistant focused on a specific topic, it is usually only necessary to search a single website or a small number of websites.  For assistants of this sort, *Custom Search Site Restricted* is a better fit since it doesn't have a limit on the number of queries that can be run per day.  Assistants that need to search more than 10 websites need to use *Custom Search* instead.
+
+## Pre-Requisite Steps
+
+- Follow the steps listed in the [Before you start](https://developers.google.com/custom-search/v1/introduction#before_you_start) section to create the custom programmable search engine (and thereafter obtain the ID of that Programmable Search Engine) as well as the [API key](https://developers.google.com/custom-search/v1/introduction#identify_your_application_to_google_with_api_key) to identify your application.
+
+## Other Setup Info
+
+- Download the OpenAPI specification (`google-custom-search-openapi.json`) from the `./basic` directory.
+- Use the OpenAPI specification to [build a custom extension](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-build-custom-extension#building-the-custom-extension).
+- [Add the extension to your assistant](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-add-custom-extension) using the API key you obtained in the pre-requisites above.
+
+### Using a template to create a new action
+- Click on `Actions` in the left menu
+- Select `Quick start with templates` to see the templates
+- Select the `Do more with starter kits` section of the template catalog
+- Select the `Google custom search starter kit` template
+- Add the template to your cart
```

Comment:
I would replace "cart" with "assistant" in this one too -- doesn't matter much though